src/cli.ts

```diff
@@ -21,7 +21,16 @@ export function run(): Promise<void> {
       type: Boolean,
     });
     var cli = commandLineArgs(argsWithHelp);
-    var cliOptions = cli.parse();
+
+    try {
+      var cliOptions = cli.parse();
```

Comment:
It does cause an exception, however it it catched in bin/polyserve (line 23). You can check the return value of a polyserve run with an unknown parameter, its 69 if there was a catched exception.
As for the test, i have to look into how to test this first.